src/MapKit/MKOverlayView.cs

```diff
@@ -15,7 +16,14 @@
 namespace MapKit {
 	public partial class MKOverlayView {
 
-#if !NET
+#if NET
+		[SupportedOSPlatform ("tvos9.2")]
+		[SupportedOSPlatform ("macos10.9")]
+		[UnsupportedOSPlatform ("ios7.0")]
+#if IOS
+		[Obsolete ("Starting with ios7.0 use 'MKOverlayRenderer' instead.", DiagnosticId = "BI1234", UrlFormat = "https://github.com/xamarin/xamarin-macios/wiki/Obsolete")]
+#endif
+#else
 		[TV (9,2)]
```

Comment:
Does this !NET section need a Deprecated ios 7?

Reply:
It gets it from the parent - https://github.com/xamarin/xamarin-macios/blob/main/src/mapkit.cs#L920